.github/workflows/reusable_build-push.yml

```diff
@@ -16,17 +16,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out GitHub Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Get current date
         id: date
-        run: echo "::set-output name=date::$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
+        run: echo "date=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_STATE
```

Comment:
see https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/